src/main/java/uk/gov/hmcts/darts/audit/service/AuditService.java

```diff
@@ -11,10 +10,7 @@
 
 public interface AuditService {
 
-    void recordAuditRequestAudio(AuditActivityEnum activity, Integer userAccountEntity, Integer courtCase);
-
-    @Transactional
-    void recordAudit(AuditActivityEnum activity, UserAccountEntity userAccountEntity, CourtCaseEntity courtCase);
+    void recordAudit(AuditActivity activity, UserAccountEntity userAccountEntity, CourtCaseEntity courtCase);
```

Comment:
It's annoying that we have a mix of parameter naming conventions in the same method
e.g. CourtCaseEntity - but it was like this before...